stdlib/map.mli

```diff
@@ -283,7 +283,24 @@ module type S =
     (** Same as {!Map.S.map}, but the function receives as arguments both the
        key and the associated value for each binding of the map. *)
 
+    (** {6 Iterators} *)
 
+    val to_seq : 'a t -> (key * 'a) Seq.t
+    (** Iterate on the whole map, in ascending order
+        @since NEXT_RELEASE *)
+
+    val to_seq_at : key -> 'a t -> (key * 'a) Seq.t
+    (** [to_seq_at k m] iterates on a subset of the bindings of [m],
+        in ascending order, from key [k] or above.
+        @since NEXT_RELEASE *)
+
+    val add_seq : 'a t -> (key * 'a) Seq.t -> 'a t
```

Comment:
`val add_seq : (key * 'a) Seq.t -> 'a t -> 'a t` for consistency with `add` and to make the partial application composable.